src/website/src/styles/components.scss

```diff
@@ -677,3 +677,18 @@ _:-ms-fullscreen,
     border: 1px solid #cccccc;
   }
 }
+
+// Utility class for an textarea element that's temporarily
+// created and used for copying content into clipboard
+.offscreen-clipboard-textarea {
```

Comment:
Is this really any different than the off-screen class we already have?

Reply:
Yeah, that's what I was intending to use. But in that class, we have these two lines which prevents selection
```
overflow: hidden !important;
visibility: hidden !important;
```